debian/control

```diff
@@ -0,0 +1,78 @@
+Source: cart
+Section: net
+Priority: optional
+Maintainer: daos-stack <daos@daos.groups.io>
+Build-Depends: debhelper (>= 10),
+               dh-python,
+               libcmocka-dev,
+               libfabric-dev,
+               libhwloc-dev,
+               libopenmpi-dev,
+               libpmix-dev,
+               libssl-dev,
+               libyaml-dev,
+               libmercury-dev,
+               scons,
+               uuid-dev
+Standards-Version: 4.1.2
+Homepage: https://wiki.hpdd.intel.com/display/DC/DAOS+Community+Home
+Vcs-Git: https://github.com/daos-stack/cart.git
+Vcs-Browser: https://github.com/daos-stack/cart.git
+
+Package: cart
+Section: net
+Architecture: any
+Multi-Arch: same
+Depends: libcart1 (= ${binary:Version}), ${shlibs:Depends}, ${misc:Depends},
+         openmpi-bin
+Description: Collective and RPC Transport
+ CaRT is an open-source RPC transport layer for Big Data and Exascale
+ HPC. It supports both traditional P2P RPC delivering and collective RPC
+ which invokes the RPC at a group of target servers with a scalable
+ tree-based message propagating.
+ .
+ This package contains the Cart Utilities.
+
+Package: libcart-dev
+Section: libdevel
+Architecture: any
+Multi-Arch: same
+Depends: libcart1 (= ${binary:Version}), ${misc:Depends}
+Description: Collective and RPC Transport
+ CaRT is an open-source RPC transport layer for Big Data and Exascale
+ HPC. It supports both traditional P2P RPC delivering and collective RPC
+ which invokes the RPC at a group of target servers with a scalable
+ tree-based message propagating.
+ .
+ This package is needed to compile programs against libcart.
+ It contains the header files and links needed for compiling.
+
+
+Package: libcart1
```

Comment:
Should be libcart0, no?  `./usr/lib/x86_64-linux-gnu/libcart.so.0.0.1`

Reply:
Indeed.  CaRT bumped up it's version from 0.0.1 to 1.0.0 since I last reviewed this.  :-)